integration/log/log_test.go

```diff
@@ -33,7 +33,20 @@ func TestLog(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	app, err := c.AppRun(context.Background(), image.ID, nil)
+	app, err := c.AppRun(context.Background(), image.ID, &client.AppRunOptions{
```

Comment:
Does this test actually verify something or you just want to enable linkerd annotation so you can manually verify the log when running in a cluster with linkerd installed?

Reply:
Yeah, I add the linkerd annotation to the app itself in this case because of the race condition that happens if I don't. The app will start running before the acorn-controller propagates the injection label from the project namespace to the app namespace, and as a result, the pods will start up without the linkerd sidecars. It's a bit weird so I left a comment in here to explain why I did it.